storage/errors.go

```diff
@@ -0,0 +1,15 @@
+package storage
+
+import "errors"
+
+var (
+	// ErrAlreadyExists may be thrown by docs. E.g., save a document twice.
+	//
+	// Since: 2.3
+	ErrAlreadyExists = errors.New("document already exists")
+
+	// ErrNotExists may be thrown by docs. E.g., save an unknown document.
+	//
+	// Since: 2.3
+	ErrNotExists = errors.New("document does not exist")
+)
```

Comment:
I realised that there already is an error in one of the other files. It would be better to have all of them in the same place

Reply:
If you are talking about the one in repository, I saw it but it is another package so I though that these errors make more sense with the storage. prefix than the repository. prefix.

Reply:
Ah, good point. Makes sense